src/main/java/com/example/demo/model/User.java

```diff
@@ -0,0 +1,176 @@
+package com.example.demo.model;
+
+
+import java.io.Serializable;
+import java.util.List;
+import java.util.Optional;
+
+import javax.persistence.CascadeType;
+import javax.persistence.ElementCollection;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+import javax.validation.constraints.Email;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+
+@Entity
+public class User implements Serializable {
+	
+	@Id
+	@GeneratedValue(strategy= GenerationType.AUTO)
+	private int id;
+	private String name;
+	private String password;
+	
+	@OneToOne(targetEntity=Address.class,cascade=CascadeType.ALL,mappedBy="user")
+	@JsonIgnoreProperties("user")
```

Comment:
is this required?

Reply:
Yes  , again recursive data of User inside address is coming as an O/P .That we do not want. So, used this annotation to ignore insertion of User properties inside address.